### PR TITLE
Fix/treasury balance over time graph 368

### LINKED ISSUE #368: Treasury balance over time graph
A new line graph representing the treasury balance over time in both EOS and USD would be a great addition to the income section or home page

### LINKED ISSUE #386: Check the margin space of the charts
**Description and expected**

We need to resize the margin spaces between the charts. This will give the user more breath when observe the information

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- Resolve #368
- Resolve #386 

### Steps to test

1. Run Project
2. Enter to Income, must see someting like this:
imagen

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

**Comment by JAMares** [comment]
This PR is "Work in progress" dont merge

**Comment by JAMares** [comment]
This PR is ready to merged

File: .github/workflows/push-dev-environment.yaml [hunk 1]
```diff
@@ -83,6 +83,7 @@ jobs:
           HAPI_DFUSE_API_KEY: ${{secrets.HAPI_DFUSE_API_KEY}}
           HAPI_DFUSE_API: jungle4.dfuse.eosnation.io
           HAPI_DFUSE_FIRST_BLOCK: ${{secrets.HAPI_DFUSE_FIRST_BLOCK}}
+          HAPI_DFUSE_FIRST_TREASURY_BLOCK: ${{secrets.HAPI_DFUSE_FIRST_TREASURY_BLOCK}}
```
Comment (xavier506) [inline .github/workflows/push-dev-environment.yaml:86]: does this need to be a secret?

Comment (JAMares) [reply .github/workflows/push-dev-environment.yaml]: Fixed

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**